src/test/resources/mocks/non-strategic/birminghamAdministrativeCourtDailyCauseList.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
These lists  are the same with the same schema. Better to avoid duplications in the test files

Reply:
using one file now